input/pagecontent/auereqdi.md

```diff
@@ -1,10 +1,345 @@
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td rowspan="4">Adverse reaction risk summary</td>
+    <td colspan="2">Substance name</td>
+    <td><a href="StructureDefinition-au-core-allergyintolerance.html">AU Core AllergyIntolerance</a></td>
+    <td>AllergyIntolerance.code</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td colspan="2">Manifestation</td>
+    <td><a href="StructureDefinition-au-core-allergyintolerance.html">AU Core AllergyIntolerance</a></td>
+    <td>AllergyInterolance.reaction</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td colspan="2">Comment</td>
+    <td><a href="StructureDefinition-au-core-allergyintolerance.html">AU Core AllergyIntolerance</a></td>
+    <td>AllergyIntolerance.note</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td colspan="2">Last updated</td>
+    <td>TBD</td>
+    <td>TBD</td>
+    <td>This data element is added to AUCDI R1; work is underway to map to AU Core.</td>
+  </tr>
+  <tr>
+    <td rowspan="5">Problem/Diagnosis summary</td>
+    <td colspan="2">Problem / Diagnosis name</td>
+    <td><a href="StructureDefinition-au-core-condition.html">AU Core Condition</a></td>
+    <td>Condition.code</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td colspan="2">Body site/laterality</td>
+    <td><a href="StructureDefinition-au-core-condition.html">AU Core Condition</a></td>
+    <td>Condition.code</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td colspan="2">Status</td>
+    <td><a href="StructureDefinition-au-core-condition.html">AU Core Condition</a></td>
+    <td>Condition.clinicalStatus</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td colspan="2">Comment</td>
+    <td><a href="StructureDefinition-au-core-condition.html">AU Core Condition</a></td>
+    <td>Condition.note</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td colspan="2">Last updated</td>
+    <td>TBD</td>
+    <td>TBD</td>
+    <td>This data element is added to AUCDI R1; work is underway to map to AU Core.</td>
+  </tr>
+    <tr>
+    <td rowspan="4">Sex and gender summary</td>
+    <td colspan="2">Sex assigned at birth</td>
+    <td><a href="StructureDefinition-au-core-patient.html">AU Core Patient</a></td>
+    <td>Patient.extension.where(url='http://hl7.org/fhir/StructureDefinition/individual-recordedSexOrGender')</td>
+    <td>The <a href="https://build.fhir.org/ig/hl7au/au-fhir-base//StructureDefinition-individual-recordedSexOrGender.html">Person Recorded Sex or Gender extension</a> is profiled by <a href="StructureDefinition-au-core-rsg-sexassignedab.html">AU Core Sex Assigned At Birth (RSG)</a> to represent the concept of Sex assigned at birth.</td>
+  </tr>
+  <tr>
+    <td colspan="2">Gender identity</td>
+    <td><a href="StructureDefinition-au-core-patient.html">AU Core Patient</a></td>
+    <td>Patient.extension.where(url='http://hl7.org/fhir/StructureDefinition/individual-genderIdentity')</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td colspan="2">Pronoun/s</td>
+    <td><a href="StructureDefinition-au-core-patient.html">AU Core Patient</a></td>
```

Comment:
The links to AU Core will need to be absolute URLs since the IG context has changed.

Reply:
AU Core URLs updated.